include/measurement_kit/README.md

```diff
@@ -365,6 +366,10 @@ These are the available options:
 - `"no_geoip"`: (boolean) whether to perform GeoIP lookup. By default set to
   `false`, meaning that we'll try;
 
+- `"no_resolver_lookup"`: (boolean) whether to lookup the IP address of
+  the resolver. By default `false`, meaning that we'll try. When true we
+  will set the resolver IP address to `127.0.0.1`;
+
```

Comment:
Is there a way, when this option is enabled, to pass to MK the IP of the resolver of the client obtained through some other means (similarly to how you can pass `probe_cc`, `probe_ip`, etc.)?

Reply:
Also should we maybe set the IP of the resolver to `127.0.0.2` so as to distinguish this scrubbed IP from the scrubbed IP of the probe (this can be useful to do a search and replace if there is lack of a proper function to pass it into MK).

Reply:
No, there is not. The reason why there is not is that the approach that I am following for probe-engine is that we disable everything and we unconditionally overwrite such fields with the values we know to be good at the end of a test.

Reply:
Ok I see. Do you see some value in using a different magic number for the resolver_ip in this case then?

Reply:
Yes. What could be a better value to use? Can we leave it empty?

Reply:
I am going to leave it empty.

Reply:
Well, actually I don't know. After this PR has landed, I will address this issue as one of the few remaining issues before blessing v0.10.5. See https://github.com/measurement-kit/measurement-kit/issues/1884.